README.md

```diff
@@ -2,7 +2,7 @@
 
 JoinMarket is software to create a special kind of bitcoin transaction called a CoinJoin transaction. Its aim is to improve the confidentiality and privacy of bitcoin transactions.
 
-A CoinJoin transaction requires other people to take part. The right resources (coins) have to be in the right place, at the right time, in the right quantity. This isn't a software or tech problem, it's an economic problem. JoinMarket works by creating a new kind of market that would allocate these resources in the best way.
+A CoinJoin transaction requires other people to participate. The right resources (coins) have to be in the right place, at the right time, in the right quantity. These are not  software or tech conditions, these are economic conditions. JoinMarket meets them by creating a new kind of market that would allocate these resources in the best way.
```

Comment:
I don't agree with the semantic change here; it is indeed a *problem* that coordination is difficult, so I'm not sure why you'd want to change from "problem", to "conditions"?